cmd/profile.go

```diff
@@ -0,0 +1,117 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"github.com/manifoldco/torus-cli/api"
+	"github.com/manifoldco/torus-cli/apitypes"
+	"github.com/manifoldco/torus-cli/config"
+	"github.com/manifoldco/torus-cli/errs"
+
+	"github.com/urfave/cli"
+)
+
+func init() {
+	profile := cli.Command{
+		Name:     "profile",
```

Comment:
We should probably also introduce a reciprocal `profile view` command that lists the email, full name, etc (only accessible by users).

We'd then make `torus status` only about the current "context" you are operating in and nothing more.